dds/DCPS/Hash.cpp

```diff
@@ -6,6 +6,7 @@
 #include "DCPS/DdsDcps_pch.h" //Only the _pch include should start with DCPS/
 
 #include "Hash.h"
+#include "Definitions.h"
```

Comment:
Is this needed?

Reply:
At one point, but not anymore... removing.